lib/isorender/dom-shims.js

```diff
@@ -84,11 +84,14 @@ Document.prototype.createElement = function(tagName) {
   return el;
 };
 
-global.customElements = global.customElements || {
+global.customElements = {
```

Comment:
I'm writing tests for mp-table, and it seems that environment has a globals.customElements which isn't bound to registeredElements. This makes the customElements (get, define) bound to the registry as expected.